crates/rooch/src/commands/session_key/mod.rs

```diff
@@ -35,3 +48,101 @@ pub enum SessionKeyCommand {
     Create(Box<CreateCommand>),
     List(ListCommand),
 }
+
+fn display_json_as_table(data: &Value) {
```

Comment:
We can make the `--json` and `--table` as an option to every command, and auto display json to table as default in the future.